docker/centos8-ortools

```diff
@@ -0,0 +1,59 @@
+FROM centos:latest
+
+# Install requirements
+RUN yum update -y &&\
+    yum install git gcc gcc-c++ make -y &&\
+    yum install wget -y &&\
+    yum install epel-release -y &&\
+    yum install unzip openssl-devel libcurl-devel wxGTK3-devel libuuid-devel boost-devel boost-test -y
+
+# Install CMake3
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.17.3/cmake-3.17.3.tar.gz &&\
```

Comment:
In centos8 cmake version is 2.8 if I remember, we must use at least 3.x.
In this case we don't use cmake3 install because version installed doesn't seems to support `cmake -B` option